frame-server/server/profile/contact_info.json

```diff
@@ -1,6 +1,10 @@
 {
+  "omit" : "ommiting any of the field below will disable them from being displayed on the Contact page",
+  "direct_phone": "XXX-XXX-XXXX", 
```

Comment:
Did you intend this to be an extra field? Seems like a comment @BlairCooper

Reply:
It is a comment but there isn't any other way to comment a JSON file.